delivery_waypoint_number: 2